cypresse2e/cypress/integration/e2eTests/signIn/signIn.spec.js

```diff
@@ -92,7 +92,7 @@ describe('FightPandemics Sign In Page', () => {
             passwordField.type('testtest');
             var passwordCharacterRequired = signIn.getPasswordCharacterRequired();
             passwordCharacterRequired.should('be.visible');
-            passwordCharacterRequired.contains('Password must contain at least 3 of these: a lower-case letter, an upper-case letter, a number, a special character (such as !@#$%^&*).');
+            passwordCharacterRequired.contains('Password must contain at least 3 of these: lower-case letter, an upper-case letter, a number, a special character (such as !@#$%^&*).');
```

Comment:
'a' should be kept to stay consistent with rest of the list of requirements. Either 'a' or 'an' should be removed for all of them or be kept for all of them.